marin/resources.py

```diff
@@ -80,7 +99,25 @@ class GpuConfig(ResourceConfig):
     runtime_env: RuntimeEnv = dataclasses.field(default_factory=RuntimeEnv)
 
     accelerator_type: AcceleratorType | None = None
-    """Type of GPU accelerator to use. If None, will use any available GPU."""
+    """Type of GPU to use. Must be one of the Ray accelerator types."""
+
+    device_flops_override: float | None = None
+    """Optional override for device FLOPS. If set, this value will be used instead of looking up in device_flops_map."""
+
+    def __post_init__(self):
+        if self.accelerator_type is not None:
+            upper = self.accelerator_type.upper()
+            if upper in _ACCEL_TYPES:
+                object.__setattr__(self, 'accelerator_type', upper)
+            else:
+                raise ValueError(
```

Comment:
This might be too strict, because Ray has to support it?  For example, if someone runs it on AMD or Tranium or something that's not supported properly, I think we just want them to be able to modify the Marin codebase and register the hardware and FLOPs? (cc @dlwh )

Reply:
with device_flops_override i think this is the right thing since accelerator_type gets passed to Ray